pkg/config/config.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/apex/log"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/urfave/cli"
-	"gopkg.in/yaml.v3"
```

Comment:
why you remove this line?

Reply:
oh sorry, my ide for some reason decided that this import is not used, restored it back.